spec.bs

```diff
@@ -232,7 +232,105 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<xmp class="idl">

```

Comment:
I'd make this `<pre class=idl>` instead of `<xmp>` as pre blocks are the standard. XMP should be used when you need to escape markup, which you're not doing here.

Reply:
I don't think this has been resolved.

Reply:
This is not resolved.